File: modules/webcam-core/src/main/java/com/shterneregen/securelan/webcam/service/impl/DefaultVideoProfileService.java
package com.shterneregen.securelan.webcam.service.impl;

import com.shterneregen.securelan.webcam.service.VideoCallProfile;
import com.shterneregen.securelan.webcam.service.VideoProfileService;

public class DefaultVideoProfileService implements VideoProfileService {
    @Override
    public VideoCallProfile defaultProfile() {
        return new VideoCallProfile(1280, 720, 30, true);
    }
}

File: modules/webcam-core/src/main/kotlin/com/shterneregen/securelan/webcam/service/impl/DefaultVideoProfileService.kt
package com.shterneregen.securelan.webcam.service.impl

import com.shterneregen.securelan.webcam.service.VideoCallProfile
import com.shterneregen.securelan.webcam.service.VideoProfileService

class DefaultVideoProfileService : VideoProfileService {
    override fun defaultProfile(): VideoCallProfile {
        return VideoCallProfile(1280, 720, 30, screenShareReady = true)
    }
}
